airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/CdcStateCompressionTest.java

```diff
@@ -36,24 +37,45 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
 import java.util.stream.Collectors;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class CdcStateCompressionTest {
+  private static final Logger LOGGER = LoggerFactory.getLogger(CdcStateCompressionTest.class);
 
   static private final String CDC_ROLE_NAME = "cdc_selector";
 
   static private final String TEST_USER_NAME_PREFIX = "cdc_test_user";
 
   static private final String TEST_SCHEMA = "test_schema";
 
-  static private final int TEST_TABLES = 10;
+  static private final int TEST_TABLES = 4;
 
+  //SQLServer tables can't have more than 1024 columns.
   static private final int ADDED_COLUMNS = 1000;
 
   private MsSQLTestDatabase testdb;
+  private final ExecutorService executor = Executors.newFixedThreadPool(10);
+  private static final String ALTER_TABLE_ADD_COLUMN_SQL;
+  static {
```

Comment:
I moved the inner loop here because it wasn't really dependent on the outer loop except for the table name that's been left as a %s here